vulnerability-management/export-workloads/README.md

```diff
@@ -0,0 +1,14 @@
+# Export workload vulnerabilities via shell script
+
+The `/v1/export/vuln-mgmt/workloads` API exports workload vulnerabilities in the form
+of deployments and their associated images including image vulnerabilities.
+
+The following sections provide use case examples utilizing either the shell or Python.
+
+## Shell script
+
+See `export-workloads.sh` for a shell script example based on `curl` and `jq`.
```

Comment:
Would be nice to have some sample invocations with some sample queries (like filtering for specific cluster or something along those lines).